test/ffmpeg-qsv/encode/av1.py

```diff
@@ -6,11 +6,97 @@
 
 from ....lib import *
 from ....lib.ffmpeg.qsv.util import *
-from ....lib.ffmpeg.qsv.encoder import AV1EncoderLPTest
+from ....lib.ffmpeg.qsv.encoder import AV1EncoderLPTest, AV1EncoderTest
 
 spec = load_test_spec("av1", "encode", "8bit")
 spec_r2r  = load_test_spec("av1", "encode", "8bit", "r2r")
 
+class cqp(AV1EncoderTest):
+  def init(self, tspec, case, gop, bframes, tilecols, tilerows,qp, quality, profile):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = gop,
+      bframes   = bframes,
+      qp        = qp,
+      rcmode    = "cqp",
+      quality   = quality,
+      profile   = profile,
+      tilerows  = tilerows,
+      tilecols  = tilecols,
+    )
+
+  @slash.parametrize(*gen_av1_cqp_lp_parameters(spec))
```

Comment:
This is the wrong parameter generator... it is getting config from the cqp_lp variant.  Please add a new generator that generates cases from the "cqp" variants.

Also, please do the same for the others (cbr, vbr).

Reply:
updated